php/nucleo/lib/salidas/toba_vista_jasperreports.php

```diff
@@ -336,7 +337,14 @@ function completar_con_datos()
 			}
 			//Creo el reporte finalmente con la conexion JDBC
 			$print = $this->jasper->fillReport($this->path_reporte, $this->parametros, $con1);
-			$con1->close();
+                    } catch (\Exception $e) {
+                        toba::logger()->error("Jasper JDBC Connection Error");
```

Comment:
```suggestion
                        toba::logger()->error('Error en la ejecución del reporte');
```
No sabes cual es la fuente del error, por eso guardas el msg.

Reply:
@enfoqueNativo Listo, corregido!!!